typings/ember/ember.d.ts

```diff
@@ -0,0 +1,3212 @@
+interface DOMElement {}
```

Comment:
What's that large new TS file for? If that's just for type hinting, I'd not commit it